update-changelog/index.ts

```diff
@@ -64,7 +64,7 @@ class UpdateChangelog extends Action {
 
 		await octokit.octokit.pulls.create({
 			base: 'main',
-			body: 'This exciting! So much has changed!',
+			body: 'This exciting! So much has changed!\nDO NOT CHANGE THE TITLES DIRECTLY IN THIS PR, everyting in the PR is auto generated.',
```

Comment:
Not sure if it's `\n` or `<br />`

Reply:
think it's `\n`, same as for comments (e.g. https://github.com/grafana/grafana-github-actions/blob/main/backport/backport.ts#L178) 👍

Reply:
```suggestion
			body: 'This exciting! So much has changed!\nDO NOT CHANGE THE TITLES DIRECTLY IN THIS PR, everything in the PR is auto-generated.',
```